R/QFeatures-class.R

```diff
@@ -630,7 +653,7 @@ setReplaceMethod("names", c("QFeatures", "character"),
 ##'
 ##' @param colvars A `character()` that selects column(s) in the
 ##'     `colData`.
-##' @param index The assay indicator for `SummarizedExperiment`
+##' @param index The assay indicator within a `SummarizedExperiment`
```

Comment:
Wouldn't it be better to refer to the index of the assat?

Reply:
Yes I fully agree, but then I would call the argument `i` instead of `index`. The latter is to comply to `MultiAssayExperiment::longFormat()`. Should we remove `index`?